Sources/APIInterface/include/WordPressComRESTAPIInterfacing.h

```diff
@@ -6,6 +6,22 @@
 
 @property (strong, nonatomic, readonly) NSURL * _Nonnull baseURL;
 
+/// Whether the user's preferred language locale should be appended to the request.
+/// Should default to `true`.
+///
+/// - SeeAlso: `localeKey` and `localeValue` to configure the locale appendend to the request.
+@property (nonatomic, readonly) BOOL appendsPreferredLanguageLocale;
+
+/// The key with which to specify locale in the parameters of a request.
+@property (strong, nonatomic, readonly) NSString * _Nonnull localeKey;
+
+/// The value with which to specify locale in the parameters of a request.
+@property (strong, nonatomic, readonly) NSString * _Nonnull localeValue;
+
+@property (strong, nonatomic, readonly) NSURLSession * _Nonnull urlSession;
+
+@property (strong, nonatomic, readonly) void (^ _Nullable invalidTokenHandler)(void);
```

Comment:
All these properties went in the protocol to make the `async perform` methods work.

The `async perform` methods, which like in `WordPressComRestApi` are Swift-only, of course.

I considered using an extension or a different protocol, but it seemed neat to have it all in `WordPressComRESTAPIInterfacing`, so that Swift code can used an upgraded version of the same interface that the Objective-C code uses. This will actually be useful in the cases where we have an ObjC superclass with the API client conforming to `WordPressComRESTAPIInterfacing` and a subclass using the API client, too.